saleor/tests/e2e/product/utils/preparing_product.py

```diff
@@ -0,0 +1,55 @@
+from . import (
+    create_category,
+    create_product,
+    create_product_channel_listing,
+    create_product_type,
+    create_product_variant,
+    create_product_variant_channel_listing,
+)
+
+
+def prepare_product(
+    e2e_staff_api_client,
+    warehouse_id=None,
+    channel_id=None,
```

Comment:
We use default values only for optional fields. In the current code warehouse and channel is required.
```suggestion
    warehouse_id,
    channel_id,
```